### PR TITLE
[CDAP-17478] assessPipeline count to ignore partially supported colum…

### PR DESCRIPTION
**Pull request by elfenheart**
…ns that are suppressed

JIRA: https://cdap.atlassian.net/browse/CDAP-17478

### CODE REVIEW

File: delta-app/src/main/java/io/cdap/delta/store/DraftService.java [hunk 1]
```diff
@@ -473,11 +473,19 @@ private TableSummaryAssessment summarize(String db, @Nullable String schema, Str
...
         numUnsupported++;
       } else if (columnAssessment.getSupport() == ColumnSupport.PARTIAL) {
-        numPartial++;
+        Set<SourceColumn> columns = table.getColumns();
+        SourceColumn existingColumn = columns.stream()
+          .filter(column -> columnAssessment.getSourceName().equals(column.getName()))
+          .findAny()
+          .orElse(null);
+
+        if (existingColumn == null || !existingColumn.isSuppressWarning()) {
```
Comment (seanzhougoogle) [inline delta-app/src/main/java/io/cdap/delta/store/DraftService.java:482]: in which case it will be null and the partial num should increase ?

Comment (elfenheart) [reply delta-app/src/main/java/io/cdap/delta/store/DraftService.java:482]: column selection is not required, so the array of column selection can be empty, in which case the column is supposed to be replicated

Comment (seanzhougoogle) [reply delta-app/src/main/java/io/cdap/delta/store/DraftService.java:482]: got it. LGTM

**Review by seanzhougoogle — APPROVED** [review]

**Comment by elfenheart** [comment]
Green build: https://builds.cask.co/browse/DEL-BT100-2

**Pull request merged**